### DISCUSSION TITLE
Having multiple shortcuts for the same operation

### DISCUSSION THREAD

**Discussion by ZhongxuanWang** [rxhanson/Rectangle#327] [Ideas]
**The feature to add more shortcuts to the same operation.**

__Eg. to move a window to 1/2 left, users can either press control+q or control+shift+q.__ 

This would help me a lot, when I would usually mix what combination of keys I should press

**Comment by rxhanson** [comment]
Thanks for the idea.

Unfortunately, this is not something I have any plans to add into Rectangle.

You can accomplish this using the custom shortcut builder in my paid app, [Hookshot](https://hookshot.app). I wrote a post about how to use it: https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24
